libmamba/include/mamba/core/mirror.hpp

```diff
@@ -1,182 +0,0 @@
-// Copyright (c) 2023, QuantStack and Mamba Contributors
```

Comment:
This file is not included by anyone, and both it and .cpp file are not mentioned in CMake, so I think it's safe to remove it.
Please, make sure I'm right